### DISCUSSION TITLE
Display day of week on X-axis

### DISCUSSION THREAD

**Discussion by mbarradas** [marcusolsson/grafana-hourly-heatmap-panel#57] [Ideas]
Current version left out the option of picking the way date were displayed. Carpet plug-in was capable of displaying the week day on the x axis which is useful.  Cheers for the great work and enhancements like being capable of picking the range of time that should be shown.

**Comment by marcusolsson** [comment]
Definitely agree that the date format on the x-axis needs more work. I'd like to incorporate localized time formats as well.

I'm following the discussion on [localizing the format for the graph panel](https://github.com/grafana/grafana/pull/25602), as it will hopefully set an example for how other panels should do this as well.
